tests/system/datastore/test_v3io.py

```diff
@@ -13,79 +13,95 @@
 # limitations under the License.
 
 import os
+import os.path
 import subprocess
 import tempfile
 import time
 import uuid
 from urllib.parse import urlparse
 
+import dask.dataframe as dd
 import pandas as pd
 import pytest
 
 import mlrun.datastore
+from mlrun.datastore.datastore_profile import (
+    DatastoreProfileV3io,
+    register_temporary_client_datastore_profile,
+)
 from tests.system.base import TestMLRunSystem
 
 
 @TestMLRunSystem.skip_test_if_env_not_configured
 @pytest.mark.enterprise
+@pytest.mark.parametrize("use_datastore_profile", [True, False])
 class TestV3ioDataStore(TestMLRunSystem):
+    test_dir = "/bigdata/v3io_tests"
+    test_dir_url = f"v3io://{test_dir}"
+    run_dir = f"{test_dir}/run_{uuid.uuid4()}"
+    profile_name = "v3io_ds_profile"
+
     @classmethod
     def setup_class(cls):
         super().setup_class()
-        cls.test_file_path = str(cls.get_assets_path() / "test.txt")
-        test_parquet_path = str(cls.get_assets_path() / "testdata_short.parquet")
-        test_csv_path = str(cls.get_assets_path() / "testdata_short.csv")
-        test_json_path = str(cls.get_assets_path() / "testdata_short.json")
+        cls.token = os.environ.get("V3IO_ACCESS_KEY")
+        assets_path = str(cls.get_assets_path())
+        cls.test_file_path = os.path.join(assets_path, "test.txt")
```

Comment:
The `cls.*` stuff can move to the class level as well.

Reply:
You can move `cls.token = os.environ.get("V3IO_ACCESS_KEY")`. You're right about the rest of them.

Reply:
resolved, tnx!

Reply:
reverted, V3IO_ACCESS_KEY is not loaded in the class level